### PR TITLE
update golang image to eu.gcr.io mirror

### PR DESCRIPTION
**Pull request by kon-angelo**
**How to categorize this PR?**

/area dev-productivity
/kind technical-debt
/priority normal
/platform openstack

**What this PR does / why we need it**:
Updates the images to the eu.grc.io golang image mirror and circumvent dockerhub rate limits

**Release note**:

```improvement operator
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ############# builder
-FROM golang:1.15.3 AS builder
+FROM eu.gcr.io/gardener-project/3rd/golang:1.15.5 AS builder
```
Comment (ialidzhikov) [inline Dockerfile:2]: Can you change also the alpine image below to `eu.gcr.io/gardener-project/3rd/alpine:3.12.1`? Just to be sure that we won't hit rate limit issue with the alpine image too.

**Review by rfranzke — APPROVED** [review]
/lgtm

**Review by dkistner — APPROVED** [review]
Thank you :)

**Pull request merged**